new-website/src/components/location/edit-form.jsx

```diff
@@ -82,12 +243,77 @@ export const EditForm = ({ docId, doc, onError }) => {
-        <TextField name="phone" label={formatMessage({ id: "partnerLocation_Phone" })} defaultValue={state.phone} onChange={handleChange} />
-        <TextField name="email" label={formatMessage({ id: "partnerLocation_Email" })} defaultValue={state.email} onChange={handleChange} />
+        <TextField name="phone" label={formatMessage({ id: "partnerLocation_Phone" })} value={state.phone} onChange={handleChange} />
+        <TextField name="email" label={formatMessage({ id: "partnerLocation_Email" })} value={state.email} onChange={handleChange} />
+      </FormGroup>
+      <FormGroup className={classes.row}>
+        <TextField name="address" label={formatMessage({ id: "partnerLocation_Address" })} value={state.address} onChange={handleChange} />
+      </FormGroup>
+      <FormGroup className={classes.row}>
+        <FormControlLabel className={classes.flexNone}
+          control={
+            <Switch
+              name="latlngExplicit"
+              color="primary"
+              checked={state.latlngExplicit}
+              onChange={handleExplLatLng}
+            />
+          }
+          label={formatMessage({ id: "partnerLocation_ExplicitPinPosition" })}
+        />
+        {state.latlngExplicit && (
+          <>
+            <TextField
+              value={explLat}
+              onChange={e => setExplLat(e.target.value)}
+              label={formatMessage({ id: "partnerLocation_Latitude" })}
+              className={classes.smallField}
+            />
+            <TextField
+              value={explLng}
+              onChange={e => setExplLng(e.target.value)}
+              label={formatMessage({ id: "partnerLocation_Longitude" })}
+              className={classes.smallField}
+            />
+          </>
+        )}
+      </FormGroup>
+      <FormGroup className={classes.row}>
+        {(state.type === "plasma") && (
+          <img src={MarkerIconPlasma} className={classes.markerIcon} alt="icon-plasma" />
+        )}
+        {(state.type === "mission") && (
+          <img src={MarkerIconMission} className={classes.markerIcon} alt="icon-mission" />
+        )}
+        {(state.type === "tafel") && (
+          <img src={MarkerIconTafel} className={classes.markerIcon} alt="icon-tafel" />
+        )}
+        {(state.type === "immunhelden") && (
+          <img src={MarkerIconImmunhelden} className={classes.markerIcon} alt="icon-immunhelden" />
+        )}
+        {(state.type === "new") && (
+          <img src={MarkerIconNew} className={classes.markerIcon} alt="icon-new" />
+        )}
+        <FormControl className={classes.smallField} style={{flex: 1}}>
+          <InputLabel id="demo-simple-select-helper-label">{formatMessage({ id: "partnerLocation_Type" })}</InputLabel>
+          <Select name="type" value={state.type} onChange={handleChange}
+                  labelId="demo-simple-select-helper-label">
+            <MenuItem value="plasma">{formatMessage({ id: "partnerLocation_TypePlasma" })}</MenuItem>
```

Comment:
do you really want to hardcode alle options and not just get them from firestore?

Reply:
I think it's fine for now.